### ISSUE TITLE
Release is not published on GitHub

### ISSUE THREAD

**Issue by clemens-tolboom** [molgenis/molgenis-service-armadillo#604]
Latest master builds do not appear on https://github.com/molgenis/molgenis-service-armadillo/tags even with #586 through PR  #603

Running `./gradlew release --info` gives

```log
...

Project previous version (nyx): v4.0.2
Corrected version checking for optional snapshot: 4.0.3
Project version : 4.0.3
Git version     : b88acec6
Git name-ref    : remotes/origin/master

...

> Task :nyxPublish
Caching disabled for task ':nyxPublish' because:
  Build cache is disabled
Task ':nyxPublish' is not up-to-date because:
  Task has not declared any outputs despite executing actions.
Publish release: v4.0.3
Resolve mutations for :release (Thread[Execution worker Thread 5,5,main]) started.
:release (Thread[Execution worker Thread 5,5,main]) started.

...
```

**Comment by clemens-tolboom** [comment]
Comparing https://github.com/molgenis/molgenis-emx2/blob/master/settings.gradle with https://github.com/molgenis/molgenis-service-armadillo/blob/master/settings.gradle

```bash
diff --side-by-side molgenis-emx2/settings.gradle molgenis-service-armadillo/settings.gradle
```

gives no hints on how to resolve this.

**Comment by clemens-tolboom** [comment]
Running `./gradlew --info --debug release` gives some info similar to above.

**Comment by clemens-tolboom** [comment]
Why did v4.0.0 0 v4.0.2 worked?

**Comment by clemens-tolboom** [comment]
This is fixed through commit 60c1ab556a2c3bc190c33b3769fb601803486c2c
